public/app/features/templating/editor_ctrl.ts

```diff
@@ -174,7 +175,7 @@ export class VariableEditorCtrl {
 
     $scope.reset = () => {
       $scope.currentIsNew = true;
-      $scope.current = variableSrv.createVariableFromModel({ type: 'query' });
+      $scope.current = variableSrv.createVariableFromModel({ type: 'query', isTemporary: true });
```

Comment:
`isTemporary` tells the constructor for this VariableType that this isn't to be stored in ReduxState. Variables are stored in ReduxState when load a dashboard and when the user clicks add in the Editor.